src/streamlink/stream/hls_playlist.py

```diff
@@ -62,6 +59,12 @@ def __init__(self):
 
 
 class M3U8Parser(object):
+    _extinf_re = re.compile("(?P<duration>\d+(\.\d+)?)(,(?P<title>.+))?")
```

Comment:
`W605 invalid escape sequence` use `r""`

Reply:
It's not from me, but OK.

Reply:
While you're touching that code you may as well fix it too :D